packages/wallets/src/components/WalletsCarouselContent/WalletsCarouselContent.tsx

```diff
@@ -27,7 +28,13 @@ const WalletsCarouselContent: React.FC = () => {
         <div className='wallets-carousel-content' ref={wallet_embla_ref}>
             <div className='wallets-carousel-content__container'>
                 {wallet_accounts_list.map(wallet => (
-                    <WalletCard key={`${wallet.loginid}`} account={wallet} />
+                    <WalletGradientBackground
+                        key={`${wallet.loginid}`}
```

Comment:
```suggestion
                        key={`wallet_gradient_background_${wallet.loginid}`}
```
go for a more unique key